### PR TITLE
Spring boot docker

### PR DESCRIPTION
**Pull request by BolsunovaNataliia**

### CODE REVIEW

**Review by vadymhrnk — APPROVED** [review]
LGTM. There are a lot of changes that are not related to Docker, so in my opinion, it would be better to make another PR for these changes

File: src/main/java/mate/academy/bookshop/model/Order.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public class Order {
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "user_id", nullable = false)
     private User user;
-    @Enumerated(EnumType.STRING)
+    @JdbcTypeCode(SqlTypes.VARCHAR)
```
Comment (vovasalyha) [inline src/main/java/mate/academy/bookshop/model/Order.java:41]: Please, don't include unrelated changes to your PRs!

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ # Final stage
+ FROM openjdk:17-jdk-slim
+ WORKDIR application
+ COPY --from=builder application/dependencies/ ./
+ COPY --from=builder application/spring-boot-loader/ ./
+ COPY --from=builder application/snapshot-dependencies/ ./
+ COPY --from=builder application/application/ ./
+ ENTRYPOINT ["java", "org.springframework.boot.loader.launch.JarLauncher"]
+ EXPOSE 8080
+
```
Comment (vovasalyha) [inline Dockerfile:17]: Don't forget about empty lines.

**Review by vovasalyha — APPROVED** [review]
Approving, but please try to avoid adding unrelated changes to PRs.

File: Dockerfile (hunk 2 above)
Comment (BolsunovaNataliia) [reply Dockerfile:17]: Ok

File: src/main/java/mate/academy/bookshop/model/Order.java (hunk 1 above)
Comment (BolsunovaNataliia) [reply src/main/java/mate/academy/bookshop/model/Order.java:41]: I will try.
